src/tests/roc_library/test_sender_receiver.cpp

```diff
@@ -11,534 +11,282 @@
 #include <stdio.h>
 #include <unistd.h>
```

Comment:
Without <stdio.h> header build fails because of `snprintf` in file `test_helpers/receiver.h`. Should I add this header directly to the receiver.h file or maybe there is some other way to fix it?

Reply:
Yeah, better move it to receiver.h because each header should be self-sufficient and should not depend on what headers the user included earlier.